javascript/javascript3/week1/index.js

```diff
@@ -90,4 +90,36 @@ myButton.addEventListener("click", () =>{
 
     })
   } 
-})
\ No newline at end of file
+})
+
+//!--Giphy App--!
+
+const gifKeyword = document.getElementById("giphy-keyword")
+const gifNumber = document.getElementById("giphy-number")
+const gifButton = document.getElementById("giphy-button")
+const gifDisplay = document.getElementById("giphy-display")
```

Comment:
Though it does not affect your app much for now, but try to develop good coding habits like uniform white-spaces all around, semicolons after finishing statements. These are super important while applying for jobs. Linting is a must in todays coding you would see.